src/character.cpp

```diff
@@ -7665,7 +7665,9 @@ bool Character::invoke_item( item *used, const std::string &method )
 
 bool Character::invoke_item( item *used, const std::string &method, const tripoint &pt )
 {
-    if( !has_enough_charges( *used, true ) ) {
+    if( !has_enough_charges( *used, true ) || ( used->is_medication() &&
+            !can_use_heal_item( *used ) ) ) {
+        add_msg_if_player( m_bad, _( "Your biology is not compatible with that healing item." ) );
```

Comment:
When try to use any item in hands (using Shift + A), get the message: "Your biology is not compatible with that healing item."  Maybe problem is here?
<img width="354" alt="1" src="https://user-images.githubusercontent.com/38557723/86120777-98a96b00-badd-11ea-92f2-c13170dac393.png">
<img width="354" alt="1" src="https://user-images.githubusercontent.com/38557723/86120794-9e06b580-badd-11ea-987f-cee5f4d7551c.png">